tests/unit/test_additions_query_generator.py

```diff
@@ -0,0 +1,72 @@
+import shutil
+from pathlib import Path
+
+from dqgen.adapters.resource_fetcher import get_file_content
+from dqgen.services.additions_query_generator import InstanceAdditionsGenerator, SimplePropertyAdditionsGenerator, \
+    ReifiedPropertyAdditionsGenerator
+
+
+def test_instance_additions_generator():
+    output_folder_path = "../test_data/output"
```

Comment:
use `tmpdir` from `pytest` to create a directory for testing purposes. https://github.com/meaningfy-ws/rdf-validator-ws/blob/master/tests/unit/test_service_layer_handers.py#L17 
after the test is run it will delete it automatically

Reply:
this shall be a constant somewhere, for example in /tests/__init__.py

Reply:
better to use pathlib to specify the location